spec/services/billing_entities/update_service_spec.rb

```diff
@@ -0,0 +1,254 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe BillingEntities::UpdateService do
+  subject(:update_service) { described_class.new(billing_entity:, params:) }
+
+  let(:billing_entity) { create(:billing_entity) }
+
+  let(:timezone) { nil }
+  let(:email_settings) { [] }
+  let(:invoice_grace_period) { 0 }
+  let(:logo) { nil }
+  let(:country) { "fr" }
+
+  let(:params) do
+    {
+      legal_name: "Foobar",
+      legal_number: "1234",
+      tax_identification_number: "2246",
+      email: "foo@bar.com",
+      address_line1: "Line 1",
+      address_line2: "Line 2",
+      state: "Foobar",
+      zipcode: "FOO1234",
+      city: "Foobar",
+      default_currency: "EUR",
+      country:,
+      timezone:,
+      logo:,
+      email_settings:,
+      billing_configuration: {
+        invoice_footer: "invoice footer",
+        document_locale: "fr",
+        invoice_grace_period:
+      }
+    }
+  end
+
+  describe "#call" do
+    it "updates the billing_entity" do
+      result = update_service.call
+
+      aggregate_failures do
```

Comment:
aggregate_failures is now applied by default. We don't need to include it. This applies to all other test in this file.

Reply:
🙏